packages/slider/test/slider.test.ts

```diff
@@ -382,6 +382,71 @@ describe('Slider', () => {
         expect(el.value).to.equal(0);
     });
 
+    it('dispatches `input` of the animation frame', async () => {
+        const inputSpy = spy();
+        const el = await fixture<Slider>(
+            html`
+                <sp-slider
+                    value="50"
+                    style="width: 100px"
+                    @input=${({ target }: Event & { target: Slider }) =>
+                        inputSpy(target.value)}
+                ></sp-slider>
+            `
+        );
+        await elementUpdated(el);
+
+        let frames = 0;
+        let shouldCountFrames = true;
+        const countFrames = (): void => {
+            frames += 1;
+            if (!shouldCountFrames) return;
+            requestAnimationFrame(countFrames);
+        };
+        requestAnimationFrame(countFrames);
+        const steps: {
+            type: 'move';
+            position: [number, number];
+        }[] = [];
+        let i = 0;
+        while (i <= 50) {
+            steps.push({
+                type: 'move',
+                position: [9 + i, 30],
+            });
+            i += 1;
+        }
+        while (i) {
+            steps.push({
+                type: 'move',
+                position: [9 + i, 30],
+            });
+            i -= 1;
+        }
+        await sendMouse({
+            steps: [
+                {
+                    type: 'move',
+                    position: [9, 30],
+                },
+                {
+                    type: 'down',
+                },
+                ...steps,
+                {
+                    type: 'up',
+                },
+            ],
+        });
+        await nextFrame();
+        shouldCountFrames = false;
+
+        expect(
+            inputSpy.callCount,
+            'should be less "input"s than frames'
+        ).to.lte(frames);
```

Comment:
As opposed to just equal? Maybe the change to the logic you suggested would allow the values to line up better and then we could do that. The other side is that the communication to the browser runner being async doesn't means you get an update to the pointer position every frame, which means less `input` events than frames, most times.